cci_nepal/analysis/dummy_data/dummy_data_modeling/model_analysis.py

```diff
@@ -0,0 +1,295 @@
+    [numeric_score_transformer(i) for i in nested] for nested in test_predictions_basic
+]
+test_predictions_items_wise_basic = list(
+    map(list, zip(*test_predictions_labels_basic))
+)  # Transforming the above list
+
+test_predictions_labels_non_basic = [
+    [numeric_score_transformer(i) for i in nested]
+    for nested in test_predictions_non_basic
+]
+test_predictions_items_wise_non_basic = list(
+    map(list, zip(*test_predictions_labels_non_basic))
+)  # Transforming the above list
+
+
+accuracy_list_basic = []
+for i in range(0, len(nfri_basic)):
+    accuracy_list_basic.append(
+        accuracy_score(test_predictions_items_wise_basic[i], y_test_basic.iloc[:, i])
+    )
+
+logging.info(
+    pd.DataFrame(accuracy_list_basic, y_test_basic.columns, columns=["Accuracy"])
+)
+
+
+accuracy_list_non_basic = []
+for i in range(0, len(nfri_non_basic)):
+    accuracy_list_non_basic.append(
+        accuracy_score(
+            test_predictions_items_wise_non_basic[i], y_test_non_basic.iloc[:, i]
+        )
+    )
+
+logging.info(
+    pd.DataFrame(
+        accuracy_list_non_basic, y_test_non_basic.columns, columns=["Accuracy"]
+    )
+)
+
+
+model_basic.fit(X, y_basic)
+model_non_basic.fit(X, y_non_basic)
+
+# Making prediction
+
+row_basic = X.iloc[
+    10,
+].tolist()
+yhat_basic = model_basic.predict([row_basic])
+
+# Summarizing prediction
+
+logging.info("Printing the scores for NFRI Basic")
+logging.info(yhat_basic[0])
+
+
+logging.info("Now printing the scores for NFRI Non Basic")
+
+
+# Making prediction
+
+row_non_basic = X.iloc[
+    10,
+].tolist()
+yhat_non_basic = model_non_basic.predict([row_non_basic])
+
+# Summarizing prediction
+
+logging.info(yhat_non_basic[0])
+
+
+# Plotting the Coefficient scores of the Model
+
+coefficient_scores = model_basic.coef_[0]
+
+for i, v in enumerate(coefficient_scores):
+    logging.info("Feature: %0d, Score: %.5f" % (i, v))
+
+
+logging.info(X.columns)
+pyplot.bar([x for x in range(len(coefficient_scores))], coefficient_scores)
+pyplot.show()
+
+coeff_basic_df = pd.DataFrame(model_basic.coef_[0], X.columns, columns=["Coefficient"])
+logging.info("The Coefficient for item", nfri_basic[0], "is:")
+coeff_basic_df
+
+# Getting the indices of items on the basis of importance in Descending order
+
+sorted_indices_basic = np.flip(np.argsort(yhat_basic[0]))
+
+sorted_indices_non_basic = np.flip(np.argsort(yhat_non_basic[0]))
+
+logging.info("NFRI Basic sorted on the basis of importance score")
+logging.info(itemgetter(*sorted_indices_basic)(nfri_basic))
+
+
+logging.info("Now time for NFRI Non Basic")
+logging.info(itemgetter(*sorted_indices_non_basic)(nfri_non_basic))
+
+logging.info("The importance label of above NFRI Basic")
+logging.info([label_transformer(d) for d in -np.sort(-yhat_basic[0])])
+
+logging.info("Now time for NFRI Non Basic")
+logging.info([label_transformer(d) for d in -np.sort(-yhat_non_basic[0])])
+
+# Evaluating multioutput regression model with k-fold cross-validation
```

Comment:
Im finding all the different outputs of results a little confusing. Might be better to collect them in variables then output them as a couple of tables at the bottom of the page. So they are more clear / easier to compare.